packages/typescriptlang-org/lib/themes/typescript-beta-light.json

```diff
@@ -2,38 +2,85 @@
   "$schema": "vscode://schemas/color-theme",
   "name": "light-plus",
   "tokenColors": [
-    { "settings": { "foreground": "#000000" } },
```

Comment:
the formatting change makes the content change hard to find. it would be easier if they were in two separate commits or even two separate PRs.